test/functionalTest/cases/3541_subscription_max_fails_limit/subscription_without_maxfailslimit_and_failscounter.test

```diff
@@ -35,8 +35,7 @@ brokerStart CB
 # 03. GET /v2/subscriptions/subscription_id, to check no maxFailsLimit and status=active
 # 04. Stop accumulator
 # 05. PATCH /v2/entities/Room1/attrs, to update the value of pressure attribute.
-# 06. GET /v2/subscriptions/subscription_id, to check no failsCounter and status=active
-# 07. POST /v2/subscriptions, to create subscription without maxFailsLimit=foo and see error
```

Comment:
Step moved to specific .test dealing with parsing error conditions (maxfailslimit_error_conditions.test)

NTC (informative)